distribution/docs/src/main/resources/_contents/_catalog-frameworks/metacard-attribute-validators-definition-contents.adoc

```diff
@@ -0,0 +1,51 @@
+==== Metacard Validator Definition
+
+Metacard Validator definitions are similar to the Validators defintions. To define Metacard Validators, your definition file must have a `metacardvalidators` key in the root object.
+
+[source,json]
+----
+{
+    "metacardvalidators": {...}
+}
+----
+
+The value of `metacardvalidators` is a map of the validator configuration parameters.
+
+[source,json]
+----
+{
+    "metacardvalidators": [{
+        "validator": "requiredattribute",
+        "key": "value"
+      }]
+}
+----
+
+[WARNING]
+====
+The configuration for the metacard validator must always contain a `validator` key in which the other
+properties are passed to an instance of that validator.
+====
+
+The `validator` key must have a value of one of the following:
+
+ - `requiredattribute` (ensures that configured set of required attributes are present on each metacard of the specified type)
+ * `metacardtype`: The metacard type in which to apply the set of required attributes.
+ * `requiredattributes`: Array of strings containing the required attributes.
+
+Examples:
+[source, json]
+----
+{
+  "metacardvalidators": [{
+    "validator": "requiredattribute",
```

Comment:
Just a thought: what if we organized metacard validators by metacard type, similar to the way we organize attribute validators by attribute?
```
{
    "metacardvalidators": {
        "fallback.common": [
            {
                "validator": "requiredattributes",
                "requiredattributes": ["id", "title", "attr"]
            }
        ]
    }
}
```

If we add more metacard validators in the future then users can easily organize their validators by metacard type.